README-WG.md

```diff
@@ -0,0 +1,62 @@
+# Kubernetes Conformance
+
+Info regarding the CNCF K8s Conformance Working Group.
+
+The Kubernetes Software Conformance Working Group operates under the aegis of CNCF. We are collaborating with the Kubernetes [Architecture](https://github.com/kubernetes/community/blob/master/sig-architecture/README.md) and [Testing](https://github.com/kubernetes/community/blob/master/sig-testing/README.md) SIGs to help ensure awareness and acceptance of our processes.
```

Comment:
Could you please update this description to describe exactly how we're collaborating with them, as per the blog post.